### PR TITLE
OKTA-519949 : Adding global css to style Header/Footer when SIW is okta hosted

### PR DESCRIPTION
**Pull request by glenfannin-okta**
## Description:

The purpose of this PR is to add a global css file for styles that are used by the loginpage_backbone.jsp when hosting the SIW. This should style the Header/Footer content, additionally, changes in this file fixes the a11y issues found in the header and footer (updates the color of the app banner paragraph content to `#111` and the footer copyright content to `#111`. 

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [x] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [x] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-519949](https://oktainc.atlassian.net/browse/OKTA-519949)

### CODE REVIEW

File: src/v3/src/components/Widget/style.module.css
Comment (glenfannin-okta) [inline src/v3/src/components/Widget/style.module.css:237]: Correct me if I am wrong but I believe this can be removed right? This content comes from the loginpagetheme.css which is actually loaded by the loginpage_backbone.jsp so no need to define them here as well. Additionally, when I test this using the embedded scenario, I can see a bunch of styles applied that should not be applied. 

cc: @lesterchoi-okta

File: src/v3/src/components/Widget/style.module.css
Comment (glenfannin-okta) [inline src/v3/src/components/Widget/style.module.css:75]: This fixes the a11y color contrast issue in the footer for the text content.

File: src/v3/src/components/Widget/style.module.css
Comment (glenfannin-okta) [inline src/v3/src/components/Widget/style.module.css:78]: This fixes the a11y color contrast issue in the footer for the link content.

File: src/v3/src/components/Widget/style.module.css
Comment (glenfannin-okta) [inline src/v3/src/components/Widget/style.module.css:115]: This fixes the a11y color contrast issue in the header for the text content.

File: src/v3/src/components/Widget/style.module.css
Comment (glenfannin-okta) [reply src/v3/src/components/Widget/style.module.css]: Nevermind, I am extremely confident this can be removed so I removed it.

File: src/v3/src/components/Widget/style.module.css
Comment (glenfannin-okta) [inline src/v3/src/components/Widget/style.module.css:79]: `color: var(--ColorTextSub);` Corrects the a11y issue in the footer (confirmed this passes axe test.

File: src/v3/src/components/Widget/style.module.css
Comment (glenfannin-okta) [inline src/v3/src/components/Widget/style.module.css:116]: This correct the a11y color contrast issue in the header.

File: src/v3/src/components/Widget/style.module.css
Comment (lkchoi) [inline src/v3/src/components/Widget/style.module.css:5]: remove comments

**Review by lkchoi — APPROVED** [review]
I notice everything in `style.module.css` had to be wrapped in `:global()`. What happens if you just name it `style.css` and import -- without the `:global()`?

**Comment by glenfannin-okta** [comment]
> I notice everything in `style.module.css` had to be wrapped in `:global()`. What happens if you just name it `style.css` and import -- without the `:global()`?

I did try this actually, when building, the output css still had the radomized characters appended to the class names: 
image

I looked for a way to globally do this, but the alternative was to make everything global, then prefix module css with something like `[local]:`. I felt like that sort of change would be more intrusive since we only need global css for this specific file while (mostly) everything else is module specific

**Comment by lkchoi** [comment]
Reminder: @shawyu-okta @leemchale-okta  needs CODEOWNERS approval

**Review by shawyu-okta — APPROVED** [review]

**Review by leemchale-okta — APPROVED** [review]

**Pull request merged**
